### PR TITLE
video and document conversions

### PR DESCRIPTION
**Pull request by art-vasilyev**
## Description

* document transformations support
* video transforamtions support

## Checklist

- [x] Tests (if applicable)
- [x] Documentation (if applicable)
- [ ] Changelog stub (or use [conventional commit messages](https://www.conventionalcommits.org/))

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -164,8 +167,71 @@ Example of using callback function for printing progress::
     ...     File.upload(fh, callback=print_progress)
     56780/56780 B
 
+
+Video conversion
+^^^^^^^^^^^^^^^^
```
Comment (dmitry-mukhin) [inline README.rst:172]:
```suggestion
^^^^^^^^^^^^^^^^

```

File: README.rst [hunk 2]
```diff
@@ -164,8 +167,71 @@ Example of using callback function for printing progress::
...
+    ]
+
+    response = video_convert_api.convert(paths)
+    video_convert_info = response.result[0]
+    converted_file = File(video_convert_info.uuid)
+
+    video_convert_status = video_convert_api.status(video_convert_info.token)
+
+Document Conversion
+^^^^^^^^^^^^^^^^^^^
```
Comment (dmitry-mukhin) [inline README.rst:203]:
```suggestion
^^^^^^^^^^^^^^^^^^^

```

File: README.rst (hunk 1 above)
Comment (art-vasilyev) [reply README.rst:172]: done

File: README.rst (hunk 2 above)
Comment (art-vasilyev) [reply README.rst:204]: done

**Review by dmitry-mukhin — APPROVED** [review]

**Pull request merged**